docs/vhlo.md

```diff
@@ -0,0 +1,132 @@
+# The VHLO Dialect
+
+## What is the VHLO Dialect?
+
+The VHLO (Versioned StableHLO) Dialect is used for serialization and stability.
+It provides a snapshot of the StableHLO dialect at a given point in time by
+versioning individual program elements
+
+VHLO is an **add-only dialect** with **versioned ops, types and attributes**,
+which means that once an op or feature is added to the dialect, it cannot be
+modified in any way that impact the semantics.
+
+Any changes to an op require a new version of an op to be added to the dialect.
```

Comment:
How about "Any changes to an op, type or attribute require a new version to be added to the dialect"?